### PR TITLE
Update master

### PR DESCRIPTION
**Pull request by blakesweeney**
Generally I try to merge dev for each release.

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (afg1) [inline .pre-commit-config.yaml:28]: Do we need an issue to track getting `poetry-lock` working properly, or is it addressed in the other pull request?

Comment (blakesweeney) [reply .pre-commit-config.yaml:28]: Let's open an issue for that

File: config/cluster.config
Comment (afg1) [inline config/cluster.config:13]: I added this to ensure e! and ENA worked this time around - we should probably review it for the next release

File: config/precompute.config
Comment (afg1) [inline config/precompute.config:7]: This change should probably be reverted

File: rnacentral_pipeline/databases/data/utils.py
Comment (afg1) [inline rnacentral_pipeline/databases/data/utils.py:31]: This is needed for import to run, but we aren't sure we want circRNA or sgRNA really. They get filtered out before sending to ensembl, but we should review what to do about these for next release

**Review by afg1 — CHANGES_REQUESTED** [review]
The only thing I think needs changing before we merge is the default release mode, which I changed to use a query on changes made in the last week. I'll revert that and then I think we're good to go.

A couple of other comments to bear in mind, but I think we can merge

File: config/precompute.config
Comment (afg1) [reply config/precompute.config]: fixed in https://github.com/RNAcentral/rnacentral-import-pipeline/pull/151/commits/0b434827c3bfb346269735a33d108a8927bdbba2

File: .pre-commit-config.yaml
Comment (afg1) [reply .pre-commit-config.yaml:28]: Issue #156 is for tracking this

File: rnacentral_pipeline/databases/data/utils.py
Comment (afg1) [reply rnacentral_pipeline/databases/data/utils.py:31]: Issue #157 is for tracking this

**Pull request merged**
